app/code/core/Mage/Tax/Model/Config.php

```diff
@@ -164,7 +164,7 @@ protected function _getStoreConfig($path, $store)
     /**
      * Check if product prices inputed include tax
      *
-     * @param   mix $store
+     * @param   mixed $store
```

Comment:
core model store?

Reply:
Not in this case .... `$store` param could be more then `Mage_Core_Model_Store` here ...

`return (bool)$this->_getStoreConfig(self::CONFIG_XML_PATH_PRICE_INCLUDES_TAX, $store);`

Btw .... other untouched methods in this class use `mixed`.